Resources/Audio/DeltaV/Voice/Vulpkanin/attributions.yml

```diff
@@ -1,29 +1,49 @@
 - files: ["dog_bark1.ogg", "dog_bark2.ogg", "dog_bark3.ogg"]
   license: "CC0-1.0"
-  copyright: "Original sound by https://freesound.org/people/abhisheky948/sounds/625497/"
+  copyright: "Original sound by abhisheky948 at https://freesound.org/people/abhisheky948/sounds/625497/"
   source: "https://freesound.org/people/abhisheky948/sounds/625497/"
 
-  files: ["dog_bark2.ogg"]
+- files: ["dog_bark2.ogg"]
   license: "CC0-1.0"
-  copyright: "Original sound by https://freesound.org/people/michael_grinnell/sounds/464400/"
+  copyright: "Original sound by michael_grinnell at https://freesound.org/people/michael_grinnell/sounds/464400/"
   source: "https://freesound.org/people/michael_grinnell/sounds/464400/"
 
-  files: ["dog_bark3.ogg"]
+- files: ["dog_bark3.ogg"]
   license: "CC0-1.0"
-  copyright: "Original sound by https://freesound.org/people/Geoff-Bremner-Audio/sounds/688201/"
+  copyright: "Original sound by Geoff-Bremner-Audio at https://freesound.org/people/Geoff-Bremner-Audio/sounds/688201/"
   source: "https://freesound.org/people/Geoff-Bremner-Audio/sounds/688201/"
 
 - files: ["dog_growl1.ogg", "dog_growl2.ogg", "dog_growl3.ogg"]
   license: "CC0-1.0"
-  copyright: "Original sound by https://freesound.org/people/Glitchedtones/sounds/372533/ - cut out three clips of dog growling, cleaned up, converted to ogg"
+  copyright: "Original sound by GlitchedTones at https://freesound.org/people/Glitchedtones/sounds/372533/ - cut out three clips of dog growling, cleaned up, converted to ogg"
   source: "https://freesound.org/people/Glitchedtones/sounds/372533/"  
 
+- files: ["dog_growl4.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Original sound taken from Paradise Station. Renamed to dog_growl4.ogg"
+  source: "https://github.com/ParadiseSS13/Paradise/blob/master/sound/goonstation/voice/growl1.ogg"
+
+- files: ["dog_growl5.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Original sound taken from Paradise Station. Renamed to dog_growl5.ogg"
+  source: "https://github.com/ParadiseSS13/Paradise/blob/master/sound/goonstation/voice/growl2.ogg"
+
+- files: ["dog_growl6.ogg"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Original sound taken from Paradise Station. Renamed to dog_growl6.ogg"
+  source: "https://github.com/ParadiseSS13/Paradise/blob/master/sound/goonstation/voice/growl3.ogg"
+
 - files: ["dog_snarl1.ogg", "dog_snarl2.ogg", "dog_snarl3.ogg"]
   license: "CC0-1.0"
-  copyright: "Original sound by https://freesound.org/people/strongbot/sounds/341090/ - cut out three clips of dog snarling, cleaned up, converted to ogg"
+  copyright: "Licensed under CC SAMPLING+ 1.0 DEED. Original sound by strongbot at https://freesound.org/people/strongbot/sounds/341090/ - cut out three clips of dog snarling, cleaned up, converted to ogg"
```

Comment:
```suggestion
  copyright: "Original sound by strongbot at https://freesound.org/people/strongbot/sounds/341090/ - cut out three clips of dog snarling, cleaned up, converted to ogg"
```
?

Reply:
Your honor, it was in the license.txt. It's been retired by creative commons. So, I'm not sure either.

Reply:
CC SAMPLING+ 1.0 DEED isnt CC0 1.0 regardless

Reply:
Upon looking at the actual link, that sound is under CC0 1.0, so it's been properly credited, otherwise, with the last commit